benchmark/opperf/tensor_operations/binary_broadcast_operators.py

```diff
@@ -62,6 +62,9 @@ def run_mx_binary_operators_benchmarks(ctx=mx.cpu(), dtype='float32', warmup=10,
                                           warmup=warmup, runs=runs)
         mx_binary_op_results += cur_op_res
 
-    # Prepare combined results for Arithmetic operators
+    # Prepare combined results for Binary Broadcast operators
     mx_binary_op_results = merge_map_list(mx_binary_op_results)
     return mx_binary_op_results
+
+
+print(run_mx_binary_broadcast_operators_benchmarks())
```

Comment:
If this is a module and not an executable, should we use logging instead of print?

Reply:
Yes. It should all be logging. I had put it as todo. Working on it:
1. logging
2. Type hints for user facing APIs

Reply:
Using logging now. All prints are removed.